mkdocs.yml

```diff
@@ -1,3 +1,66 @@
-site_name: SIG Multicluster 
+site_name: SIG Multicluster
+repo_url: https://github.com/lauralorenz/sig-multicluster-site-proposal
+repo_name: lauralorenz/sig-multicluster-site-proposal
+site_dir: site-src
+docs_dir: site-src
 theme:
   name: material
+  icon:
+    repo: fontawesome/brands/git-alt
+  logo: images/k8s-favicon.png
+  favicon: images/k8s-favicon.png
+  features:
+    - search.highlight
+    - navigation.tabs
+    - navigation.top
+edit_uri: edit/main/site-src/
+plugins:
+  - search
+  - awesome-pages
+  - macros:
+      include_dir: examples
```

Comment:
I think this one got lost somewhere (this config line crashes when I run it locally) -- did these get moved to `apis/`?